hera_qm/utils.py

```diff
@@ -610,3 +613,66 @@ def strip_extension(path, return_ext=False):
         return (root, ext[1:])
     else:
         return os.path.splitext(path)[0]
+
+
+def apply_yaml_freq_time_flags(uv, a_priori_flag_yaml, lat_lon_alt_degrees=None, telescope_name='HERA'):
```

Comment:
Does it also make sense to apply antenna flags using this same function? I know it'd be redundant in xrfi, but I think it would be useful for internal logical consistency and future usefulness of this function.

Reply:
I agree. We might as well add that here.